### ISSUE TITLE
0.0.0-dev.26 issues resolving crypto, fs, path

### ISSUE THREAD

**Issue by alexisbmills** [chase-moskal/webp-hero#40]
I'm in the process of updating [ngx-webp-polyfill](https://github.com/turnstileweb/ngx-webp-polyfill) to dev-26, and have run into the following issues errors when running the app (Angular CLI webpack).

```
WARNING in ./node_modules/webp-hero/libwebp/dist/webp.js
Module not found: Error: Can't resolve 'crypto' in '<my_omitted_path>\node_modules\webp-hero\libwebp\dist'

ERROR in ./node_modules/webp-hero/libwebp/dist/webp.js
Module not found: Error: Can't resolve 'fs' in '<my_omitted_path>\node_modules\webp-hero\libwebp\dist'

ERROR in ./node_modules/webp-hero/libwebp/dist/webp.js
Module not found: Error: Can't resolve 'path' in '<my_omitted_path>\node_modules\webp-hero\libwebp\dist'
```

It seems the new version of https://github.com/chase-moskal/webp-hero/blob/master/libwebp/dist/webp.js has dependencies on the following NodeJS libraries (not available in the browser):
- crypto
- fs
- path

Is there some way around this?

**Comment by alexisbmills** [comment]
After digging a bit deeper it seems like it's an Angular issue, which may be resolved by patching some webpack config.

**Comment by chase-moskal** [comment]
@alexisbmills 

i believe the error is this one: https://stackoverflow.com/questions/59487224/webpack-throws-error-with-emscripten-cant-resolve-fs

i've found the source of the regression [in this commit](https://github.com/chase-moskal/webp-hero/commit/e4e0c9f64d45e1a557aa4e90b5634e31e7fdf6df#diff-94ebb415a3d4537ec8a4047ee7ea38d9L548)

the work to fix this in webp-hero would be:
1. add environment setting to that cmakelists file
2. run the docker build of libwebp
3. run webp-hero build with `npm i -s`
4. verify webp-hero still works with `npm start` and visit `http://localhost:5000/?force`, inspect an image to check it's converted to png base64

not sure if i'll have time to do this, i'd be happy to accept a pull request

**Comment by chase-moskal** [comment]
@alexisbmills i think i've fixed this issue in `webp-hero@0.0.0-dev.27`

**Comment by alexisbmills** [comment]
Great, I'll check it shortly.
